### DISCUSSION TITLE
Arp-Scan "IP Match" is using the public/internet domain name

### DISCUSSION THREAD

**Discussion by meeowgler** [netalertx/NetAlertX#689] [General]
The Arp-Scan "IP Match" appears to be using the public/internet domain name. For internal devices, this creates an incorrect device name. Is there a way to use the DNS search domain or the device's PTR domain?

**Comment by jokob-sk** [comment]
Hey, this guide might help:

https://github.com/jokob-sk/NetAlertX/blob/main/docs/REVERSE_DNS.md
